package.json

```diff
@@ -19,7 +19,8 @@
       "tests/addon-template"
     ],
     "nohoist": [
-      "**/@types/broccoli-plugin"
+      "**/@types/broccoli-plugin",
+      "ember-data-latest"
```

Comment:
This exists to work around a yarn issue, where yarn accidentally collapses `ember-data@<the latest version>` and `ember-data-latest` into the same node_modules entry.